backend/store/etcd/health_store.go

```diff
@@ -10,14 +11,15 @@ import (
 )
 
 // GetClusterHealth retrieves the cluster health
-func (s *Store) GetClusterHealth(ctx context.Context) *types.HealthResponse {
+func (s *Store) GetClusterHealth(ctx context.Context, cluster clientv3.Cluster, etcdClientTLSConfig *tls.Config) *types.HealthResponse {
```

Comment:
FYI this method could definitely moved from the store; it doesn't use the store's client anymore. Not sure if it's worth doing now.